docs/proposals/optional_application_loging.md

```diff
@@ -0,0 +1,106 @@
+Optional Application Logging
+============================
+
+The EFK aggregated logging stack by default collects application logs with
+fluentd from STDOUT of each container managed by OpenShift. Currently only way
+of opting out is based on a node label and ansible variable
+`openshift_logging_fluentd_nodeselector` as described in the
+[documentation](https://docs.openshift.org/latest/install_config/aggregate_logging.html).
+This proposal discusses some ways of a finer grained level of control over
+which container logs are stored in Elasticsearch.
+
+
+## Fluentd pipeline
+
+The container logs can be obtained from one of these supported logging drivers
+- json-file - logs read from `/var/log/containers/*` mounted in fluentd
+              container
+- journald
+
+Latest development shifted recommended docker driver from journald to json-file,
+therefore, the proposed implementation has focused on usage of json-file as
+well. Rationale being, journald currently doesn't match performance expectations
+and desired throttling control for larger clusters.
+
+Each container on the host has its own json structured log file
+- `/var/log/containers/[pod]_[namespace]_[container]-[docker_id].log`
+
+Every line is treated as a separate log message and processed in the cascade
+of fluentd filters with currently only supported sink Elasticsearch.
```

Comment:
`.... of fluentd filters.`
I believe we can remove ref to Elasticsearch as the final sync is immaterial of opt-in logging